### PR TITLE
[#961] refactor(coordinator):  Improve Coordinator Log Format.

### PR DESCRIPTION
**Pull request by slfan1989**
### What changes were proposed in this pull request?

While reading the code and completing the PR, I came across some log messages in the coordinator module that use string concatenation. We are already using slf4j, and it would be better to use placeholder syntax.

### Why are the changes needed?

log format.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

(Please test your changes, and provide instructions on how to test it:
  1. If you add a feature or fix a bug, add a test to cover your changes. 
  2. If you fix a flaky test, repeat it for many times to prove it works.)

### CODE REVIEW

File: coordinator/src/main/java/org/apache/uniffle/coordinator/web/servlet/DecommissionServlet.java [hunk 1]
```diff
@@ -42,9 +42,7 @@ protected Response<Object> handlePost(HttpServletRequest req, HttpServletRespons
       return Response.fail("Parameter[serverIds] should not be null!");
     }
     ClusterManager clusterManager = coordinator.getClusterManager();
-    params.getServerIds().forEach((serverId) -> {
-      clusterManager.decommission(serverId);
-    });
+    params.getServerIds().forEach(clusterManager::decommission);
```
Comment (zuston) [inline coordinator/src/main/java/org/apache/uniffle/coordinator/web/servlet/DecommissionServlet.java:45]: Emm... This is not related with this issue. Right?

Comment (slfan1989) [reply coordinator/src/main/java/org/apache/uniffle/coordinator/web/servlet/DecommissionServlet.java:45]: Thank you very much for your help in reviewing the code! This change does not belong to this issue, I will fix it.

**Comment by slfan1989** [comment]
@jerqi Can you help review this pr? Thank you very much!

**Review by jerqi — APPROVED** [review]
LGTM, thanks @slfan1989 @zuston , merged to master.
